tests/e2e/tests/electron/critical-path/files-auto-update/enablement-area-autoupdate.e2e.ts

```diff
@@ -32,15 +32,15 @@ if (fs.existsSync(workingDirectory)) {
     tutorialsTimestampFile.save();
 
     fixture `Auto-update in Enablement Area`
-        .meta({ type: 'critical_path', rte: rte.standalone })
+        .meta({ type: 'critical_path', rte: rte.standalone, skipComment: "Skipped because it is not running in CI" })
```

Comment:
I'm wondering if we can add "automatic" skip when tests are running in CI. I was thinking for something like:
```
if (process.env.CI === "true") {
    console.log("Running in CI environment");
}

test.skipIfCI = process.env.CI ? test.skip : test;

test.skipIfCI("test something cool here", () => {
    expect(true).toBe(true);
});
```

maybe it's not for this PR, so treat it as just an idea. The problem that bugs me is that someone manually has to go and remove `.skip` when running these tests locally = 100% chance something would be forgotten.